src/argilla/client/feedback/schemas.py

```diff
@@ -142,6 +306,41 @@ def update_settings(cls, values: Dict[str, Any]) -> Dict[str, Any]:
 
 
 class _LabelQuestion(QuestionSchema):
+    """A label question schema for a feedback dataset.
+
+    Args:
+        name: The name of the question.
+        title: The title of the question.
+        description: The description of the question.
+        required: Whether the question is required or not.
+        settings: The settings of the question.
+        labels: The labels of the label question.
+        visible_labels: The number of visible labels of the label question.
+
+    Examples:
+        >>> import argilla as rg
+        >>> question = rg.LabelQuestion(
+        ...     name="relevant",
+        ...     title="Is the response relevant for the given prompt?",
+        ...     description="Select all that apply",
+        ...     required=True,
+        ...     settings={"type": "label_selection"},
+        ...     labels=["cat-1", "cat-2"],
+        ...     visible_labels=None
+        ... )
+        >>> # or use a dict
+        >>> question = rg.LabelQuestion(
+        ...     name="relevant",
+        ...     title="Is the response relevant for the given prompt?",
+        ...     description="Select all that apply",
+        ...     required=True,
+        ...     settings={"type": "label_selection"},
```

Comment:
This is not intended to be done by the users, done automatically, so I would remove it from here since this may cause confusion